ui/desktop/app/templates/scopes/scope/authenticate.hbs

```diff
@@ -57,19 +57,5 @@
       </p>
 
     {{/if}}
-    {{#unless @model.authMethods}}
```

Comment:
Should we actually keep this? What if someone for whatever reason just deleted all their auth methods like this 😆 

<img width="1601" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/6e2bb585-0d79-4612-a276-b2a95f0177e7">

Reply:
Yeah that is a good point. I will add it back now.